### PR TITLE
Make Wallet Reloadable component wait on all wallet waiting keys

### PR DESCRIPTION
**Pull request by akalin-keybase**

### CODE REVIEW

File: shared/constants/wallets.js [hunk 1]
```diff
@@ -449,30 +449,32 @@ export const updatePaymentsReceived = (
...
-export const getDisplayCurrencyWaitingKey = (id: Types.AccountID) => `wallets:getDisplayCurrency:${id}`
-export const linkExistingWaitingKey = 'wallets:linkExisting'
-export const loadEverythingWaitingKey = 'wallets:loadEverything'
-export const buildPaymentWaitingKey = 'wallets:buildPayment'
-export const sendPaymentWaitingKey = 'wallets:stellarSend'
-export const requestPaymentWaitingKey = 'wallets:requestPayment'
-export const setAccountAsDefaultWaitingKey = 'wallets:setAccountAsDefault'
-export const deleteAccountWaitingKey = 'wallets:deleteAccount'
+const p = 'wallets:'
+export const acceptDisclaimerWaitingKey = `${p}acceptDisclaimer`
```
Comment (chrisnojima) [inline shared/constants/wallets.js]: i believe this ruins the flow typing

Comment (akalin-keybase) [reply shared/constants/wallets.js]: How so? Flow still succeeds...

Comment (chrisnojima) [reply shared/constants/wallets.js]: in the past flow got confused by this vs static strings.maybe its fixed, maybe not. it also depends on how it looks from outside the file. aka is this exported as a string or the constant etc

Comment (akalin-keybase) [reply shared/constants/wallets.js]: Okay, changed it back.

**Comment by akalin-keybase** [comment]
PTAL

**Comment by chrisnojima** [comment]
Yeah we shouldn't black bar due to rpc with this

**Comment by akalin-keybase** [comment]
I'll do the black bars in a separate PR -- per our chat today, seems complicated enough to warrant a separate review.

**Pull request merged**